build.sbt

```diff
@@ -6,6 +10,31 @@ version := "0.2-SNAPSHOT"
 
 scalaVersion := "2.11.8"
 
+
+SbtScalariform.scalariformSettings
+
+ScalariformKeys.preferences := ScalariformKeys.preferences.value
+    .setPreference(SpacesWithinPatternBinders, true)
+    .setPreference(SpaceBeforeColon, false)
+    .setPreference(SpaceInsideParentheses, false)
+    .setPreference(SpaceInsideBrackets, false)
+    .setPreference(SpacesAroundMultiImports,true)
+    .setPreference(PreserveSpaceBeforeArguments, false)
+    .setPreference(CompactStringConcatenation,false)
+    //.setPreference(NewlineAtEndOfFile, false)
+    .setPreference(DanglingCloseParenthesis,Force)
+    .setPreference(CompactControlReadability, false)
+    .setPreference(AlignParameters, false)
+    .setPreference(AlignArguments, true)
+    .setPreference(AlignSingleLineCaseStatements, false)
+    .setPreference(DoubleIndentClassDeclaration, false)
+    //.setPreference(DoubleIndentMethodDeclaration, true)
```

Comment:
Maybe we could consider to remove those commented lines and add all preferences in a document with an explanation why we are removing those lines.